Backpack/Calendar/Classes/BPKCalendarAppearance.h

```diff
@@ -1,108 +1,21 @@
-/*
- * Backpack - Skyscanner's Design System
- *
- * Copyright 2018 Skyscanner Ltd
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *   http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
+//
+//  BPKCalendarAppearance.h
+//  Backpack
+//
+//  Created by Aron Szabados on 2018. 11. 16..
```

Comment:
The Backpack license should be kept here.
There are a few others that are wrong too

Reply:
Headers fixed